### PR TITLE
Finished?

### PR DESCRIPTION
**Pull request by Vinc97**
So do you like the layout of this?

### CODE REVIEW

File: client/main.html [hunk 1]
```diff
@@ -17,18 +17,11 @@
   {{#if currentUser}}
   <div class="formular">{{> addPlayerForm}}</div>
   <ul>
-    {{#each player}}
-    <li class="player {{selectedClass}}">{{name}}: {{score}}</li>
-
+    {{#each player in players}}
+    <li class="player {{selectedClass}}">
+      {{> PlayerCard currentPlayer=player}}
```
Comment (mbatchkarov) [inline client/main.html:22]: The name `currentPlayer` is a bit too long for my taste. In the context of a `PlayerCard` you will have to refer to it as `this.currentPlayer`, which is redundant- a card has just one player. I would prefer something like `player`

File: client/main.html [hunk 2]
```diff
@@ -17,18 +17,11 @@
   {{#if currentUser}}
   <div class="formular">{{> addPlayerForm}}</div>
   <ul>
-    {{#each player}}
-    <li class="player {{selectedClass}}">{{name}}: {{score}}</li>
-
+    {{#each player in players}}
+    <li class="player {{selectedClass}}">
```
Comment (philkoenig) [inline client/main.html:21]: still has list bullets

File: client/main.html [hunk 3]
```diff
@@ -37,7 +30,24 @@
...
+      <input class="input-group-field" type="text" name="playerName" placeholder="Max Mustermann">
+      <div class="input-group-button">
+        <input type="submit" class="button" value="Add Player">
+      </div>
+    </div>
   </form>
+</template>
+
+<template name="PlayerCard">
+  <div class="playerCard">
```
Comment (philkoenig) [inline client/main.html:43]: buttons are not in line with the name (no vertical align)

File: client/main.html (hunk 2 above)
Comment (mbatchkarov) [inline client/main.html:21]: Using `<li>` leaves a dot next to the card, which is ugly. Better to use a `div` and use css grid to arrange cards in a list-like structure on screen

Comment (philkoenig) [inline client/main.html:21]: why is selected Class still here?

File: client/main.html [hunk 4]
```diff
@@ -37,7 +30,24 @@
 
 <template name="addPlayerForm">
   <form>
-    <input type="text" name="playerName">
-    <input class="button" type="submit" value="Add Player">
+    <div class="input-group">
```
Comment (philkoenig) [inline client/main.html:33]: well done

**Review by mbatchkarov — COMMENTED** [review]
Looking much nicer than before, nice!. I'd consider changing the colour of those buttons- the red/green/blue is too much in my face

File: client/styles/app.css [hunk 5]
```diff
@@ -1,24 +1,49 @@
...
+  margin: 0.5rem;
+}
 .formular {
-  width: 250px;
+  width: 30rem;
 }
-.content {
-  margin: 2%;
+p {
+  color: red;
```
Comment (philkoenig) [inline client/styles/app.css:11]: don't use "red" "blue" etc. use hex codes or, as you did well below, the rgb function. also, dont create color palets like a childrens birthday

**Review by philkoenig — CHANGES_REQUESTED** [review]
good progress and well done (copying ;) ) the stuff. now it's about making things look "nice" and interact well

**Pull request closed without merging**